src/handlers/store-standardised-invoices/field-utils/get-number-from-money-text.test.ts

```diff
@@ -0,0 +1,35 @@
+import { getNumberFromMoneyText } from "./get-number-from-money-text";
+
+describe("Number from money text getter", () => {
+  test.each`
+    givenText        | expectedResult
+    ${"0.00"}        | ${0}
+    ${"10000.00"}    | ${10_000}
+    ${"10,000.00"}   | ${10_000}
+    ${"£0.00"}       | ${0}
+    ${"£10000.00"}   | ${10_000}
+    ${"£10,000.00"}  | ${10_000}
+    ${"£ 0.00"}      | ${0}
+    ${"£ 10000.00"}  | ${10_000}
+    ${"£ 10,000.00"} | ${10_000}
```

Comment:
Aren't there also prices with the comma and decimal point switched (e.g. "£10.000,00") or does that only happen with quantities?